<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticeSendWdg</class>
 <widget class="QWidget" name="NoticeSendWdg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>发布公告</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="vCLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(50, 50, 50);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget#listWidget{
	border: 0px;
	background-color: rgb(67, 67, 67);
}

QListWidget#listWidget:item{
	width:320px;
	border: 1px solid #434343;
	border-radius: 0px;
	background-color: rgb(45, 45, 45);
}

</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_send" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_send{
background-color: rgb(52,54,58);	
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>32</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>-1</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QTextEdit {	
font: 12px &quot;微软雅黑&quot;;
	border: 1px solid rgb(196, 196, 196);
	border-top-left-radius : 4px;
	border-bottom-left-radius: 4px;
    background: white;
	padding-left: 10px;
	padding-top: 2px;
    color: rgb(90, 90, 90);
}
 
QTextEdit:hover {	
	border: 1px solid rgb(153, 153, 153);
  	border-top-left-radius : 4px;
	border-bottom-left-radius: 4px; 
    background: white;
    color: rgb(254, 99, 106);
}

QTextEdit:focus {	
	border: 1px solid rgb(153, 153, 153);
 	border-top-left-radius : 4px;
	border-bottom-left-radius: 4px;
    background: white;
	padding-left: 10px;
    color: rgb(90, 90, 90);
}

</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>请输入公告~</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_send">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>32</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{	
	font: 12px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:0px;
	border-top-right-radius :4px;
	border-bottom-right-radius:4px;
	background-color: rgb(166, 166, 168);
}

QPushButton:hover{
	font: 12px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:0px;
	border-top-right-radius :4px;
	border-bottom-right-radius:4px;
	background-color: rgb(150, 150, 150);
}

QPushButton:disable{
	font: 12px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:0px;
	border-top-right-radius :4px;
	border-bottom-right-radius:4px;	
	background-color: rgb(180, 180, 180);
}

</string>
        </property>
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../VhallLive/project/VhallLive.qrc"/>
  <include location="../../Project/MainUI.qrc"/>
 </resources>
 <connections/>
</ui>
